app/javascript/packs/tta-report.js

```diff
@@ -0,0 +1,11 @@
+document.querySelector("#js-add-tta-report").addEventListener('click', event => {
+  const form = document.querySelector("#tta-activity-form")
+  form.classList.remove("display-none")
```

Comment:
[suggestion] could we automatically focus on the new form when this is opened?

Reply:
heck yeah we can